functests/common.sh

```diff
@@ -135,17 +122,14 @@ function check_drives_status() {
 }
 
 function add_drives() {
-    url=$(minikube service --namespace=directpv admin-service --url)
-    admin_server=${url#"http://"}
-
     config_file="$(mktemp)"    
 
-    if ! "${DIRECTPV_CLIENT}" discover --admin-server "${admin_server}" --output-file "${config_file}"; then
+    if ! "${DIRECTPV_CLIENT}" discover --output-file "${config_file}"; then
         echo "$ME: error: failed to discover the devices"
         rm "${config_file}"
         return 1
     fi
-    if ! echo Yes | "${DIRECTPV_CLIENT}" init "${config_file}" --admin-server "${admin_server}"; then
+    if ! echo Yes | "${DIRECTPV_CLIENT}" init "${config_file}"; then
```

Comment:
Do we still need `Yes` confirmation?

Reply:
Yes, we need it. We prompt for the confirmation before proceeding.